internal/compiler/v3/compiler_v3.go

```diff
@@ -120,6 +120,9 @@ func (c *CompilerV3) getVariables(t *taskfile.Task, call *taskfile.Call, evaluat
 	if err := t.Vars.Range(taskRangeFunc); err != nil {
 		return nil, err
 	}
+	if err := t.Env.Range(taskRangeFunc); err != nil {
+		return nil, err
+	}
```

Comment:
I am not sure if the order is correct, or if I should call this before `t.Vars` and `call.Vars`